extensions/2.0/Khronos/KHR_materials_sheen/README.md

```diff
@@ -0,0 +1,158 @@
+# KHR\_materials\_sheen
+
+## Khronos 3D Formats Working Group
```

Comment:
Please change this line to `## Contributors` and add a contributors list similar to the one in the [XMP extension README](https://github.com/KhronosGroup/glTF/blob/8832254c547c642cddbd4e4b1e70aedd4eb1434e/extensions/2.0/Khronos/KHR_xmp/README.md).

Also, please copy the Khronos full copyright statement appendix from that extension.  Thanks!